test/e2e/fixtures/then.go

```diff
@@ -185,6 +186,38 @@ func (t *Then) ExpectAuditEvents(filter func(event apiv1.Event) bool, num int, b
 	return t
 }
 
+func (t *Then) ExpectPVCDeleted() *Then {
+	t.t.Helper()
+	timeout := defaultTimeout
+	_, _ = fmt.Println("Checking", timeout.String(), "for expecting PVCs deletion")
+	ctx, cancel := context.WithTimeout(context.Background(), timeout)
```

Comment:
would it make sense to use [`time.After`](https://pkg.go.dev/time#After) instead since the context is not really used here?

Reply:
Sorry I don't get it. The context is used in the following for loop for pvcClient to get PVCs: https://github.com/argoproj/argo-workflows/pull/10424/files#diff-75dd759badaa0fcc320bb8cac2c095d3585bb87b257fe4fe967279d6b21cf830R204

Reply:
My bad I missed the use of `ctx` and you only used the `context.WithTimeout` to have a timeout.

I though that we could replace: 

```
case <-ctx.Done():
```

with 

```
case <-time.After(defaultTimeout):
```

and remove the creation but missed the use of `ctx`.